docs/testing/runnning-tests.md

```diff
@@ -1 +0,0 @@
--- TO BE DEFINED --
```

Comment:
I removed this file since running tests is architecture specific and it would be better described in architecture specific documents. So I have covered this by adding a `Running tests` section to the general document about automated testing (https://github.com/openeuropa/openeuropa/blob/OPENEUROPA-590/docs/automated-testing.md#running-tests) and a Drupal specific section (https://github.com/openeuropa/openeuropa/blob/OPENEUROPA-590/docs/architecture/third-party/drupal/testing.md#running-tests).